deps/src/main/java/com/microsoft/azure/sdk/iot/deps/ws/impl/WebSocketUpgrade.java

```diff
@@ -18,6 +18,7 @@
 public class WebSocketUpgrade
 {
     public static final String RFC_GUID = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
+    @SuppressWarnings("FieldCanBeLocal")
```

Comment:
There are just some constants defined to help build the URI. This field is only used in one method but could possibly be used somewhere else later on.